app/controllers/line_items_controller.rb

```diff
@@ -0,0 +1,32 @@
+class LineItemsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_line_item, only: [:update, :destroy, :show]
+
+  def add_to_cart
+    order = current_user.cart
+    quantity = params[:line_item][:quantity] || 1
```

Comment:
This will allow 0, since `0 || 1` will be 0.

If you want to make sure the quantity is above 0, then add a validation to the `line_item` model.